GitCommands/RevisionReader.cs

```diff
@@ -459,9 +423,14 @@ public StringLineReader(in ReadOnlySpan<char> s)
                     ? bodySlice.ToString().Replace('\v', '\n')
                     : null;
             }
-        }
 
-        #endregion
+            void ParseAssert(string message)
+            {
+                _noOfParseError++;
+                Debug.Assert(!Debugger.IsAttached || _noOfParseError > 1, message);
```

Comment:
`_noOfParseError > 1` comes from the original code. Though why was it `> 1`? 

```suggestion
                Debug.Assert(!Debugger.IsAttached, message);
```

Reply:
To not have too many popups at parsing errors